test/e2e/apps/deployment.go

```diff
@@ -40,13 +40,18 @@ import (
 	deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/replicaset"
-	testutil "k8s.io/kubernetes/test/utils"
+	testutils "k8s.io/kubernetes/test/utils"
```

Comment:
this change is technically not needed?
the other imports end up on `...util` instead of `...utils`

Reply:
TL;DR - cosmetic changes, should be their own commit/PR bundled together with similar changes.

Reply:
you are right, i just saw `testutils` in a bunch of other places so i figured I would change it here as well. But I'll do all that in a following commit/pr then